packages/core/src/browser/diff-uris.ts

```diff
@@ -71,16 +71,22 @@ export class DiffUriLabelProviderContribution implements LabelProviderContributi
 
     getName(uri: URI): string {
         const [left, right] = DiffUris.decode(uri);
+        let title;
+        if (uri.path.toString() !== left.path.toString()) {
+            title = `${uri.displayName}: `;
+        } else {
+            title = '';
+        }
 
-        if (left.path.toString() === right.path.toString() && left.query && right.query) {
-            return `${left.displayName}: ${left.query} ⟷ ${right.query}`;
+        if (uri.path.toString() === right.path.toString() && left.query && right.query) {
+            return `${title}${left.query} ⟷ ${right.query}`;
         } else {
             const leftLongName = this.labelProvider.getName(left);
             const rightLongName = this.labelProvider.getName(right);
             if (leftLongName === rightLongName) {
                 return leftLongName;
             }
-            return `${leftLongName} ⟷ ${rightLongName}`;
+            return `${title}${leftLongName} ⟷ ${rightLongName}`;
```

Comment:
@benoitf can you explain this change, please?

this looks odd:

<img width="1297" alt="Screen Shot 2019-04-24 at 09 43 39" src="https://user-images.githubusercontent.com/914497/56641444-b0819580-6675-11e9-9d81-e7f11d024e1e.png">

Reply:
in VS Code diff command, it displays the displayName of the diff URL:
```
DIFF : sc:a <--> sc:b
```

Reply:
without that change I see only `sc:a <--> sc:b` in tab title

Reply:
maybe I could remove the title on the else switch

Reply:
Please check the condition from L75, shouldn't paths be equal in my example?

cf. in vscode
<img width="684" alt="Screen Shot 2019-04-24 at 09 55 05" src="https://user-images.githubusercontent.com/914497/56642406-a496d300-6677-11e9-9e8f-f781770c6111.png">

Reply:
@AlexTugarev could you tell me the steps you're doing ?

Reply:
Yes, sure.
1. compare your branch with the `master` branch
2. double click `plugin-vscode-commands-contribution.ts`

![2019-04-24 10 10 38](https://user-images.githubusercontent.com/914497/56643254-8fbb3f00-6679-11e9-872d-ee1d3029473f.gif)

Reply:
ok thanks, will fix

Reply:
ok I've changed the code:

![image](https://user-images.githubusercontent.com/436777/56649480-7fa95c80-6685-11e9-8c84-dac59f902785.png)